Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM golang:1.13.9-alpine3.11 as builder
+
+WORKDIR /go/github.com/doitintl/kube-no-trouble
+
+COPY go.mod .
+COPY go.sum .
+COPY scripts scripts
+
+RUN scripts/alpine-setup.sh 
+
+RUN go mod download
+
+COPY . .
+
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 make all
```

Comment:
just `make all` is sufficient to avoid duplication, the other bits are in the Makefile/auto-detected

Reply:
sure will change